src/nemo_evaluator/engine/model_traffic_log.py

```
@@ -4,7 +4,7 @@
 
 import time
 from datetime import datetime, timezone
-from typing import Any
+from typing import Any, Iterable, Iterator
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Iterable`, `Iterator`

Import from `collections.abc`

(UP035)